BridgeAppSDK/SBAUser.swift

```diff
@@ -43,8 +43,8 @@ public final class SBAUser: NSObject, SBAUserWrapper {
     
     let lockQueue = DispatchQueue(label: "org.sagebase.UserLockQueue", attributes: [])
 
-    public func logout() {
-        lockQueue.async {
+    public func resetStoredUserData() {
+        lockQueue.sync {
```

Comment:
Doesn't it access the user state via the lockQueue, though, ensuring that this will finish first?